pkg/cli/cmd/radinit/recipe_test.go

```diff
@@ -17,243 +17,110 @@ limitations under the License.
-				"recipes/local-dev/mongodatabases",
-				"recipes/local-dev/unsupported",
-				"recipes/unsupported/rediscaches",
-				"recipes/unsupported/unsupported",
-			},
-			"latest",
-			map[string]map[string]corerp.RecipePropertiesClassification{
-				"Applications.Datastores/redisCaches": {
-					"default": &corerp.BicepRecipeProperties{
-						TemplateKind: to.Ptr(recipes.TemplateKindBicep),
-						TemplatePath: to.Ptr(fmt.Sprintf("%s/recipes/local-dev/rediscaches:latest", DevRecipesRegistry)),
-					},
-				},
-				"Applications.Datastores/mongoDatabases": {
-					"default": &corerp.BicepRecipeProperties{
-						TemplateKind: to.Ptr(recipes.TemplateKindBicep),
-						TemplatePath: to.Ptr(fmt.Sprintf("%s/recipes/local-dev/mongodatabases:latest", DevRecipesRegistry)),
-					},
-				},
-			},
-		},
-		{
-			"Valid Prod and Dev Repositories with Redis Cache, Mongo Database",
-			[]string{
-				"recipes/local-dev/rediscaches",
-				"recipes/local-dev/mongodatabases",
-				"dev/recipes/local-dev/rediscaches",
-				"dev/recipes/local-dev/mongodatabases",
+			"Mongo Database Dev Recipe",
+			args{
+				*NewDevRecipe(
+					"mongodatabases",
+					ds_ctrl.MongoDatabasesResourceType,
+					RecipeRepositoryPrefix+"mongodatabases",
+				),
+				"0.20",
 			},
-			"latest",
-			map[string]map[string]corerp.RecipePropertiesClassification{
-				"Applications.Datastores/redisCaches": {
-					"default": &corerp.BicepRecipeProperties{
-						TemplateKind: to.Ptr(recipes.TemplateKindBicep),
-						TemplatePath: to.Ptr(fmt.Sprintf("%s/recipes/local-dev/rediscaches:latest", DevRecipesRegistry)),
-					},
-				},
-				"Applications.Datastores/mongoDatabases": {
-					"default": &corerp.BicepRecipeProperties{
-						TemplateKind: to.Ptr(recipes.TemplateKindBicep),
-						TemplatePath: to.Ptr(fmt.Sprintf("%s/recipes/local-dev/mongodatabases:latest", DevRecipesRegistry)),
-					},
+			map[string]corerp.RecipePropertiesClassification{
+				"default": &corerp.BicepRecipeProperties{
+					TemplateKind: to.Ptr(recipes.TemplateKindBicep),
+					TemplatePath: to.Ptr(RecipeRepositoryPrefix + "mongodatabases:0.20"),
 				},
 			},
 		},
 	}
-
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			if got := processRepositories(tt.repos, tt.tag); !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("processRepositories() = %v, want %v", got, tt.want)
-			}
-		})
-	}
-}
-
-func Test_isDevRepository(t *testing.T) {
-	tests := []struct {
-		name string
-		repo string
-		want bool
-	}{
-		{
-			"Dev Repository",
-			"dev/recipes/local-dev/rediscaches",
-			true,
-		},
-		{
-			"Prod Repository",
-			"recipes/local-dev/rediscaches",
-			false,
-		},
-	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := isDevRepository(tt.repo); got != tt.want {
-				t.Errorf("isDevRepository() = %v, want %v", got, tt.want)
+			if got := getRecipeProperties(tt.args.devRecipe, tt.args.tag); !reflect.DeepEqual(got, tt.want) {
```

Comment:
Please use `require.Equal` for this. We have embraced using assertion libraries in this project.